test/promql/tests/data_plane/latency_snapshot_to_static_integration_plr_creation_test.yaml

```diff
@@ -0,0 +1,99 @@
+evaluation_interval: 1m
+
+rule_files:
+  - 'prometheus.latency_snapshot_to_static_integration_plr_creation_alerts.yaml'
+
+tests:
+  - interval: 1m
+    input_series:
+      # Simulating data from Cluster 1 (crosses the 10% threshold)
+      - series: 'integration_svc_snapshot_created_to_pipelinerun_with_static_env_started_seconds_bucket{le="5", source_cluster="cluster01"}'
+        values: '0+10x10'  # 10 requests took less than 5s
```

Comment:
The comment might be a bit misleading. It's 10 requests per minute (same goes for other comments).

Reply:
It's ready to be merged, but I don't have issues with updating the comments

Reply:
updated PTAL